RatS/filmtipset/filmtipset_ratings_parser.py

```diff
@@ -0,0 +1,82 @@
+import os
+import datetime
+import time
+import fileinput
+import re
+from shutil import copyfile
+from RatS.base.base_ratings_downloader import RatingsDownloader
+from RatS.filmtipset.filmtipset_site import Filmtipset
+from RatS.utils import file_impex
+
+# Request download form
```

Comment:
- [ ] remove this comment, as how the form looks like in the HTML is not relevant and quickly outdated